generator/template.go

```diff
@@ -195,6 +195,97 @@ function getNotifyEntityArrivalSink<T>(notifyCallback: NotifyStreamEntityArrival
     }
   })
 }
+
+type RequestPayload = Record<string, unknown>;
+type Primitive = string | boolean | number;
+
+/**
+ * Checks if given value is of a primitive type
+ * @param  {unknown} value
+ * @return {boolean}
+ */
+function isPrimitive(value: unknown): boolean {
```

Comment:
`["string", "number", "boolean"].some(t => typeof value === t)`

Reply:
Thank you for this simpler implementation suggestion 🙇🏼